python/paddle/fluid/tests/unittests/CMakeLists.txt

```diff
@@ -503,7 +503,7 @@ function(bash_test_modules TARGET_NAME)
   cmake_parse_arguments(bash_test_modules "${options}" "${oneValueArgs}"
                         "${multiValueArgs}" ${ARGN})
 
-  set(timeout 350)
+  set(timeout 500)
```

Comment:
It's too long.

Reply:
Done. It is bug in TCPStore, for we always wait for 0.5s even receive the right response